src/timer.cc

```diff
@@ -77,9 +81,10 @@ void Timer::DoBuild() {
 }
 
 void Timer::DoTick() {
-  for (std::map<int, TimeListener*>::iterator agent = tickers_.begin();
-       agent != tickers_.end();
-       agent++) {
+  #pragma omp parallel for
+  for (int i = 0; i < tickers_.size(); i++) {
```

Comment:
Is changing this to an integer loop necessary for the OMP prama?

Reply:
I think so, it will not compile using the previous method of iteration.  OpenMP only supports `random_access_iterator`s as a loop variable, but `std::map` only supports `bidirectional_iterator`s (per this [forum](https://stackoverflow.com/questions/8848870/use-openmp-in-iterating-over-a-map)).  If we wanted to keep the loop as is I think we would need to copy the contents of `tickers_` to a `std::vector` (or something else that supports `random_access_iterator`s) and then iterate over it.